package/linux-mainline/package

```diff
@@ -3,14 +3,11 @@
 # SPDX-License-Identifier: MIT
 
 archs=(rm2)
-pkgnames=(linux-mainline)
-pkgdesc="reMarkable 2 kernel based on the mainline kernel"
-url=https://www.kernel.org
+pkgnames=(linux-mainline linux-mainline-perf)
 pkgver=6.0.0-1
```

Comment:
version bump?

Reply:
I don't think it's necessary to do a version bump, since the built `linux-mainline` and `linux-stracciatella` packages are unchanged, and the `*-perf` packages are new.

Reply:
I thought that we used the last part in the version number to track changes to the build system. Either way works for me.

Reply:
We should use version bumps whenever we need users to receive a new version of a package. The last part is useful when there are changes to the package but the upstream version has not changed.